website/content/docs/services/discovery/dns-forwarding.mdx

```diff
@@ -0,0 +1,323 @@
+server=/consul/127.0.0.1#8600
+
+# Uncomment and modify as appropriate to enable reverse DNS lookups for
+# common netblocks found in RFC 1918, 5735, and 6598:
+#rev-server=0.0.0.0/8,127.0.0.1#8600
+#rev-server=10.0.0.0/8,127.0.0.1#8600
+#rev-server=100.64.0.0/10,127.0.0.1#8600
+#rev-server=127.0.0.1/8,127.0.0.1#8600
+#rev-server=169.254.0.0/16,127.0.0.1#8600
+#rev-server=172.16.0.0/12,127.0.0.1#8600
+#rev-server=192.168.0.0/16,127.0.0.1#8600
+#rev-server=224.0.0.0/4,127.0.0.1#8600
+#rev-server=240.0.0.0/4,127.0.0.1#8600
+```
+
+</CodeBlockConfig>
+
+Restart the `dnsmasq` service after creating the configuration.
+
+You can configure additional settings in `dnsmasq` that may be useful in your environments. See
+[`dnsmasq(8)`](http://www.thekelleys.org.uk/dnsmasq/docs/dnsmasq-man.html)
+for additional details.
+
+<CodeBlockConfig filename="/etc/dnsmasq.conf">
+
+```plaintext
+# Accept DNS queries only from hosts whose address is on a local subnet.
+#local-service
+
+# Don't poll /etc/resolv.conf for changes.
+#no-poll
+
+# Don't read /etc/resolv.conf. Get upstream servers only from the command
+# line or the dnsmasq configuration file (see the "server" directive below).
+#no-resolv
+
+# Specify IP address(es) of other DNS servers for queries not handled
+# directly by consul. There is normally one 'server' entry set for every
+# 'nameserver' parameter found in '/etc/resolv.conf'. See dnsmasq(8)'s
+# 'server' configuration option for details.
+#server=1.2.3.4
+#server=208.67.222.222
+#server=8.8.8.8
+
+# Set the size of dnsmasq's cache. The default is 150 names. Setting the
+# cache size to zero disables caching.
+#cache-size=65536
+```
+
+</CodeBlockConfig>
+
+</Tab>
+
+<Tab heading="Unbound" group="unbound">
+
+<Note>
+
+ If your distribution uses systemd, you may need to disable `systemd-resolved` prior to following this example. 
+ 
+</Note>
+
+[Unbound](https://www.unbound.net/) is typically configured in a `unbound.conf` file or a series of files in
+the `/etc/unbound/unbound.conf.d` directory. Add the following settings to your Unbound configuration file
+(e.g., `/etc/unbound/unbound.conf.d/consul.conf`):
+
+<CodeBlockConfig filename="/etc/unbound/unbound.conf.d/consul.conf">
+
+```plaintext
+#Allow insecure queries to local resolvers
+server:
+  do-not-query-localhost: no
+  domain-insecure: "consul"
+
+#Add consul as a stub-zone
+stub-zone:
+  name: "consul"
+  stub-addr: 127.0.0.1@8600
+```
+
+</CodeBlockConfig>
+
+You may have to add the following line to the bottom of your
+`/etc/unbound/unbound.conf` file for the new configuration to be included.
+
+<CodeBlockConfig filename="/etc/unbound/unbound.conf">
+
+```plaintext
+include: "/etc/unbound/unbound.conf.d/*.conf"
+```
+
+</CodeBlockConfig>
+
+</Tab>
+
+<Tab heading="iptables" group="iptables">
+
+The rules must be set on the same host as the Consul instance. Relay hosts _should not_ be on the same host, otherwise the redirects will intercept the traffic.
```

Comment:
```suggestion
You must set the rules on the same host as the Consul instance. Relay hosts _should not_ be on the same host, otherwise the redirects intercept the traffic.
```

- changed active to passive 
- present tense